### PR TITLE
finish sdk refactor

### PR DESCRIPTION
**Pull request by wow-sven**
## Summary

Summary about this PR

- Closes #issue

### CODE REVIEW

**Review by TwilightLogic — APPROVED** [review]

File: sdk/typescript/rooch-sdk/server.log
Comment (geometryolife) [inline sdk/typescript/rooch-sdk/server.log:1]: 这个文件需要跟踪？

Comment (wow-sven) [reply sdk/typescript/rooch-sdk/server.log:1]: 不需要，已经删除了

**Comment by wow-sven** [comment]
@yubing744 Please review it. It's basically finished

**Comment by jolestar** [comment]
```
Notice: Git status is dirty
  xargs: warning: options --max-args and --replace/-I/-i are mutually exclusive, ignoring previous --max-args value
  Notice:   File: frameworks/rooch-framework/doc/auth_payload.md
  Notice:   File: frameworks/rooch-framework/doc/bitcoin_validator.md
  Notice:   File: sdk/typescript/rooch-sdk/src/client/types/generated.ts
  Notice:   File: sdk/typescript/rooch-sdk/src/client/types/params.ts
```

**Comment by yubing744** [comment]
This PR https://github.com/rooch-network/rooch/pull/2016/checks E2E test failed, but Pipeline did not fail

**Comment by yubing744** [comment]
The unit test coverage is too low, it is recommended to be greater than 90%

**Pull request merged**